tests/common

```diff
@@ -1,12 +1,21 @@
 
 # shellcheck shell=bash
 
-SWTPM=swtpm
-SWTPM_EXE=${SWTPM_EXE:-${ROOT}/src/swtpm/${SWTPM}}
-SWTPM_IOCTL=${SWTPM_IOCTL:-${ROOT}/src/swtpm_ioctl/swtpm_ioctl}
-SWTPM_BIOS=${SWTPM_BIOS:-${ROOT}/src/swtpm_bios/swtpm_bios}
-SWTPM_SETUP=${SWTPM_SETUP:-${ROOT}/src/swtpm_setup/swtpm_setup}
-SWTPM_CERT=${SWTPM_CERT:-${ROOT}/src/swtpm_cert/swtpm_cert}
+if [ -z "${INSTALLED}" ]; then
+    SWTPM=swtpm
+    SWTPM_EXE=${SWTPM_EXE:-${ROOT}/src/swtpm/${SWTPM}}
+    SWTPM_IOCTL=${SWTPM_IOCTL:-${ROOT}/src/swtpm_ioctl/swtpm_ioctl}
+    SWTPM_BIOS=${SWTPM_BIOS:-${ROOT}/src/swtpm_bios/swtpm_bios}
+    SWTPM_SETUP=${SWTPM_SETUP:-${ROOT}/src/swtpm_setup/swtpm_setup}
+    SWTPM_CERT=${SWTPM_CERT:-${ROOT}/src/swtpm_cert/swtpm_cert}
+else
+    SWTPM_EXE=${SWTPM_EXE:-$(type -P swtpm)}
+    SWTPM_IOCTL=${SWTPM_IOCTL:-$(type -P swtpm_ioctl)}
+    SWTPM_BIOS=${SWTPM_BIOS:-$(type -P swtpm_bios)}
+    SWTPM_SETUP=${SWTPM_SETUP:-$(type -P swtpm_setup)}
+    SWTPM_CERT=${SWTPM_CERT:-$(type -P swtpm_cert)}
+fi
```

Comment:
All of these could go into the test runner and the changes to this file could be removed.

Reply:
kinda, it's unfortunate not to be able to run the test individually easily.. Now you have to run them with INSTALLED=1, which is kinda ok until we have a better solution...alternatively, we generate a different file for installation. wdyt?

Reply:
There's test_config.in that could be sourced by all tests and contain INSTALLED=0|1 (maybe edited with sed when installing?)

Reply:
See last patch for a nicer alternative.

Reply:
LGTM. Call variable `SWTPM_TEST_UNINSTALLED`?

Reply:
done